channels/us.m3u

```diff
@@ -6620,19 +6620,19 @@ https://streams.the6tv.duckdns.org:2443/locals/SoCal/kabc-dt2.m3u8
 https://597f64b67707a.streamlock.net/almagd.tv/almagd.smil/playlist.m3u8
 #EXTINF:-1 tvg-id="" tvg-name="" tvg-country="" tvg-language="Arabic" tvg-logo="https://www.almagd.tv/en/images/logo.png" group-title="Religious",Almagd TV North America
 https://5aafcc5de91f1.streamlock.net/almagd.tv/almagd.smil/playlist.m3u8
-#EXTINF:-1 tvg-id="LocalNowHarrisburgPA.us" tvg-name="LocalNow - Harrisburg PA" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Harrisburg PA
+#EXTINF:-1 tvg-id="LocalNowHarrisburgPA.us" tvg-name="LocalNow - Harrisburg PA" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Harrisburg PA [Geo blocked]
 https://prod.localnowapi.com/vod/api/v2/LiveStream/Playback/paHarrisburg/index.m3u8
-#EXTINF:-1 tvg-id="LocalNowPhiladelphiaPA.us" tvg-name="LocalNow - Philadelphia PA" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Philadelphia PA
+#EXTINF:-1 tvg-id="LocalNowPhiladelphiaPA.us" tvg-name="LocalNow - Philadelphia PA" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Philadelphia PA [Geo blocked]
 https://prod.localnowapi.com/vod/api/v2/LiveStream/Playback/paPhiladelphia/index.m3u8
-#EXTINF:-1 tvg-id="LocalNowNewYorkNY.us" tvg-name="LocalNow - New York City NY" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow New York City NY
+#EXTINF:-1 tvg-id="LocalNowNewYorkNY.us" tvg-name="LocalNow - New York City NY" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow New York City NY [Geo blocked]
 https://prod.localnowapi.com/vod/api/v2/LiveStream/Playback/nyNewYorkCity/index.m3u8
-#EXTINF:-1 tvg-id=".us" tvg-name="LocalNow - Phoenix AZ" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Phoenix AZ
+#EXTINF:-1 tvg-id=".us" tvg-name="LocalNow - Phoenix AZ" tvg-country="US" tvg-language="English" tvg-logo="https://i.imgur.com/x8KRwBz.png" group-title="Local",LocalNow - Phoenix AZ [Geo blocked]
```

Comment:
1. Leave tvg-id empty. It will be filled with .us part automatically (unless you want to specify call signs)
2. Geoblocked tag is spelled with - in it [Geo-blocked]. That matters for some checks

Reply:
Okay, I changed that all ready. So empty the call sign area? I didn't think those channels have call signs, they jump all over the place. From The Weather Channel to local news channels.

Reply:
There's no need to leave that .us part in there. Otherwise it won't be filled by bot